src/components/templates/PosterPage/components/PosterPageSettingsControl/PosterPageSettingsControl.tsx

```diff
@@ -1,45 +1,70 @@
-import React from "react";
+import React, { useCallback } from "react";
 import { OverlayTrigger, Popover } from "react-bootstrap";
 import { faCog } from "@fortawesome/free-solid-svg-icons";
 
+import { setVenueLiveStatus } from "api/venue";
+
 import { Checkbox } from "components/atoms/Checkbox";
 
 import { PosterPageControl } from "../PosterPageControl";
 
 import "./PosterPageSettingsControl.scss";
 
 export interface PosterPageSettingsControlProps {
-  setVenueLiveOff: () => void;
-  setVenueLiveOn: () => void;
+  venueId: string;
   isPosterLive?: boolean;
 }
 
 export const PosterPageSettingsControl: React.FC<PosterPageSettingsControlProps> = ({
-  isPosterLive,
-  setVenueLiveOff,
-  setVenueLiveOn,
+  isPosterLive = false,
+  venueId,
 }) => (
   <OverlayTrigger
     trigger="click"
     placement="bottom-end"
     overlay={
-      <Popover
-        id="poster-page-settings-popover"
-        className="PosterPageSettingsControl__popover"
-      >
-        <Popover.Content className="PosterPageSettingsControl__popover-content">
-          <Checkbox
-            type="checkbox"
-            checked={!!isPosterLive}
-            onChange={isPosterLive ? setVenueLiveOff : setVenueLiveOn}
-            title={isPosterLive ? "Poster is live" : "Make poster live"}
-            toggler
-          />
-        </Popover.Content>
-      </Popover>
+      <PosterPageSettingsControlOverlay
+        isPosterLive={isPosterLive}
+        venueId={venueId}
+      />
     }
     rootClose={true}
   >
     <PosterPageControl label="Settings" icon={faCog} />
   </OverlayTrigger>
 );
```

Comment:
As called out in my new comments on the previous review thread about refactoring this overlay content into a standalone component, according to the docs (referenced in the other comment thread), it seems that `OverlayTrigger` requires it's rendered child to be capable of receiving a `ref`, etc?

Reply:
Answered in the same thread above